src/test/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMessageHandlerTest.java

```diff
@@ -195,4 +200,80 @@ void handleInboundMeshMessageWithMultipleMessagesAndPublishesToGpOutboundQueue()
             () -> verify(meshOutboundQueueService).publish(outboundMeshMessage)
         );
     }
+
+    @Test
+    void handleInboundMeshMessageWithNoNhsAckRequestedAndPublishesToGpOutboundQueue()
+            throws InterchangeParsingException, MessageParsingException {
+
+        Interchange interchangeNoAckRequested = mock(Interchange.class);
+
+        when(interchangeNoAckRequested.getInterchangeHeader()).thenReturn(
+                InterchangeHeader.builder()
```

Comment:
Should be 4 and not 8 spaces. Check other places.

Reply:
Should be fixed now.